src/tests/Microsoft.PowerFx.Interpreter.Tests.Shared/RuntimeErrorTests.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Globalization;
+using System.Linq;
+using Microsoft.PowerFx.Types;
+using Xunit;
+
+namespace Microsoft.PowerFx.Core.Tests
+{
+    public class RuntimeErrorTests
+    {
+        [Fact]
+        public void RuntimeErrorLocalizedTests()
+        {
+            var engine = new RecalcEngine();
+            var check = new CheckResult(engine)
+                .SetText("1/0")
+                .SetBindingInfo()
+                .SetDefaultErrorCulture(CultureInfo.CreateSpecificCulture("pt-BR"));
```

Comment:
>  .SetDefaultErrorCulture [](http://example.com/codeflow?start=14&length=25)

ErrorCulture is a misnomer - this is really for design time errors via LSP, so would be better named "LSP culture". 

For runtime, we should be using the *runtime culture*.

[https://github.com/microsoft/Power-Fx/blob/main/src/libraries/Microsoft.PowerFx.Interpreter/Environment/SymbolExtensions.cs#L33](https://github.com/microsoft/Power-Fx/blob/main/src/libraries/Microsoft.PowerFx.Interpreter/Environment/SymbolExtensions.cs#L33)